src/assets/icon/IconCheck.js

```diff
@@ -1,17 +1,37 @@
 import * as React from 'react';
-import Svg, {Path, Rect} from 'react-native-svg';
-import {COLORS} from '../../utils/theme';
+import PropTypes from 'prop-types';
+import Svg, {G, Path, Defs, ClipPath, Rect} from 'react-native-svg';
 
-const IconCheck = (props) => (
-  <Svg width={32} height={32} fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
-    <Rect width={32} height={32} rx={16} fill={COLORS.signed_primary} />
-    <Path
-      fillRule="evenodd"
-      clipRule="evenodd"
-      d="M23.699 10.285a1 1 0 0 1 .016 1.414l-9.765 10a1 1 0 0 1-1.479-.053l-4.234-5a1 1 0 0 1 1.526-1.292l3.524 4.16 8.998-9.213a1 1 0 0 1 1.414-.016Z"
-      fill={COLORS.white}
-    />
-  </Svg>
-);
+function IconCheck(props) {
+  return (
+    <Svg
+      viewBox="0 0 24 24"
+      width={props?.width ?? 20}
+      height={props?.height ?? 20}
+      fill={props?.fill}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of optional chaining (`props?.width`) and nullish coalescing operator (`?? 20`) is a good practice to provide default values for the `width`, `height`, and `fill` properties. However, it's important to note that if `0` is passed as a value, it will be replaced by the default value due to the behavior of the nullish coalescing operator. If you want to allow `0` as a valid value, consider using a different approach.

```diff
- width={props?.width ?? 20}
- height={props?.height ?? 20}
+ width={props.width !== undefined ? props.width : 20}
+ height={props.height !== undefined ? props.height : 20}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->